package.json

```diff
@@ -23,18 +23,23 @@
     "build:product-details": "webpack --env entryPattern=./src/tests/product-details/*.test.js",
     "build:product-management": "webpack --env entryPattern=./src/tests/product-management/*.test.js",
     "build:product-search": "webpack --env entryPattern=./src/tests/product-search/*.test.js",
```

Comment:
add tags for existing tests and use entityTag dynamicly:

npm run build --tags:smoke,checkout (builds only checkout tests that have smoke tag)
npm run build --tags:soak (builds all tests that has soak tag)

+ get rid scripts sections started with `build:...`